anitya/tests/db/test_models.py

```diff
@@ -260,6 +360,79 @@ def test_packages__repr__(self):
         self.assertEqual(str(pkg), '<Packages(1, Fedora: geany)>')
 
 
+class ProjectFlagTestCase(DatabaseTestCase):
+    """ Tests for ProjectFlag model. """
+
+    def test_project_flag__repr__(self):
+        """ Test the ProjectFlag.__repr__ function. """
+        flag = create_flagged_project(self.session)
+
+        self.assertEqual(str(flag), '<ProjectFlag(geany, dgay@redhat.com, open)>')
```

Comment:
The test name implies it's testing ``__repr__``, shouldn't this call ``repr(flag)``?

Reply:
Probably, my bad.